nginx-app/templates/default/redirect-ssl.conf.erb

```diff
@@ -9,6 +16,6 @@ server {
     ssl_prefer_server_ciphers on;
     server_name <%=@domain_name%>;
 <% unless @new_domain_name.nil? -%>
-    rewrite ^ https://<%=@new_domain_name%>$request_uri? permanent;
+    rewrite ^ <%=@new_domain_name%> permanent;
```

Comment:
I removed this, and already update the stack json on QA. So it should be streamlined and can be used for both redirects.